src/node/node_state.h

```diff
@@ -984,6 +986,13 @@ namespace ccf
       verify_snapshot_end_unsafe();
     }
 
+    void advance_part_of_public_network()
+    {
+      std::lock_guard<std::mutex> guard(lock);
+      history->start_signature_emit_timer();
```

Comment:
Missing an expect (ReadingPublicLedger, I'm guessing, but please check)

Reply:
Added this, and local runs passed. Will let the CI stress the full test suite before merging.